eng/testing/tests.singlefile.targets

```diff
@@ -34,6 +39,10 @@
     <SelfContained>true</SelfContained>
   </PropertyGroup>
 
+  <PropertyGroup Condition="">
+    <DefineConstants>$(DefineConstants);SINGLE_FILE_TEST_RUN</DefineConstants>
```

Comment:
What's the purpose of adding the <code>SINGLE_FILE_TEST_RUN</code> property? That should only be needed for conditional C# compilation, do we really use that? Also please note that the pre-existing bit at line 5 you deleted used a slightly different name for the macro, <code>SINGLE_FILE_TEST_RUNNER</code>, can you please clarify or double-check if we're consistent here?

Reply:
Wow that was a typo! It's supposed to be `SINGLE_FILE_TEST_RUNNER` there, not `SINGLE_FILE_TEST_RUN`. Thanks for the catch.